docs/configuration.md

```diff
@@ -12,8 +12,10 @@ Lighter can be configured by using environment variables. Currently, Lighter sup
 | LIGHTER_EXTERNAL_LOGS_URL_TEMPLATE     | Template for link to external logs (Grafana, Graylog, etc.) used on the Lighter UI. Allowed placeholders: `{{id}}`, `{{appId}}`, `{{createdTs}}` |                                 |
 | LIGHTER_PY_GATEWAY_PORT                | Port for live Spark session communication                                                                                                        | 25333                           |
 | LIGHTER_URL                            | URL which can be used to access Lighter form Spark Job                                                                                           | http://lighter.spark:8080       |
-| LIGHTER_SESSION_TIMEOUT_MINUTES        | Session lifetime in minutes (from last statement creation). Use negative value to disable                                                        | 90                              |
-| LIGHTER_SESSION_TIMEOUT_ACTIVE         | Should Lighter kill sessions with waiting statements (obsolete when `LIGHTER_SESSION_TIMEOUT_MINUTES` is negative)                               | false                           |
+| LIGHTER_SESSION_TIMEOUT_INTERVAL       | `java.time.Duration` representing session lifetime (from last statement creation). Use `0` value to disable                                      | 90m                             |
```

Comment:
‘0’ as a default value would probably wont work for Java Duration?

Reply:
I did change the documentation to use `0m`. Although what's strange, that it should work with `0` also, but at least in groovy tests I got `NULL` value instead of Duration.ZERO which I expected and which I get with `0m`. I might be wrong but [documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.conversion.durations) states that using plain number should default to milliseconds.